hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -233,10 +233,12 @@
 
     <list name="default">
         <backup-count>1</backup-count>
+        <merge-policy batch-size="100">PutIfAbsentMergePolicy</merge-policy>
```

Comment:
Can you add it to `hazelcast-fullconfig.xml` as well?

Reply:
Fixed